### DISCUSSION TITLE
Touring mode

### DISCUSSION THREAD

**Discussion by DavidWeijzen** [organicmaps/organicmaps#3000] [Ideas]
Hello,

When I ride my motorcycle, I would love to use the twistiest roads possible while still getting from point A to point B. Just disabling highways doesn't do it since there are usually straight provintial roads right next to it to provide transportation for vehicles who can't go on the highway. As far as I am aware, the only apps who do this are proprietary apps.
Perhaps it could be implemented with a slider for how much delay you're willing to take for the sake of twistyness.

**Comment by Esmolante** [comment]
I think the algo doesn't take account of the geometry of the road, only generic max speed. The route with that delay could be a detour with more kilometers and not necessarily more curves. 
It's interesting, because the solution to find a route with many curves would be the same as to avoid them, optimizing the arrival time.
Let's see what the bosses have to say.
Gazzzzzzzzzz!

**Comment by biodranik** [comment]
Alternate routes can solve this issue, but there should be clear algorithmic criteria for it.
